lib/big_brother/node.rb

```diff
@@ -30,7 +30,7 @@ def incorporate_state(another_node)
     end
 
     def invalidate_weight!
-      @weight = nil
+      @weight = 0
```

Comment:
Why is it better to set the weight to 0 rather than nil, isn't 0 a valid weight?